server/src/catalog.rs

```diff
@@ -155,20 +117,18 @@ impl Catalog {
         // HACK: workaround for DataFusion converting table names to lowercase.
         let normalized_file_or_folder_name = file_name.to_ascii_lowercase();
```

Comment:
Will this cause troubles if I have two tables named "Table" and "table"? Or is that not possible?

Reply:
Fixed by adding `Catalog::normalize_table_name()` which currently converts all table names to lowercase UTF-8 so paths containing UTF-8 can be used and to make comparisons case insensitive.